eng/doc/CrossPlatformCryptography.md

```diff
@@ -100,19 +100,19 @@ This section includes the following packages:
 
 <sup>1</sup>When using OpenSSL 3, requires the legacy provider to be enabled.
 
-### AES-GCM keys, nonces, and tags
+<sup>2</sup> AES-GCM supports specific keys, nonces, and tags:
 
-* Key Sizes
+  * Key Sizes
```

Comment:
intentional indenting here?

Reply:
It was, but it doesn't seem to actually do anything (and the normal bullet indent looks fine to me on GitHub), so I'll remove.